tuta-sdk/rust/sdk/src/crypto/aes.rs

```diff
@@ -395,15 +401,16 @@ fn aes_encrypt<Key: AesKey>(
 		PaddingMode::WithPadding => encryptor.encrypt_padded_vec_mut::<Pkcs7>(plaintext),
 	};
 
+	let iv_and_ciphertext = join_slices!(iv.0.as_slice(), &encrypted_data);
```

Comment:
I think originally I wanted to avoid copying it when it's not necessary but if I understand it right we always need it anyway

Reply:
Is this something we need to change? We decided to merge only the web and sdk parts...

Reply:
nah I think @paw-hub and @vaf-hub already looked at it